public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.js

```diff
@@ -644,3 +665,8 @@ class LibreTextsGlossarizer {
 
     }
 }
+
+
+//Self Initialize
+let libretextGlossary = new LibreTextsGlossarizer();
+libretextGlossary.makeGlossary();
```

Comment:
I would also recommend surrounding these lines with an eventListener. This also prevents this variable from leaking into the global scope (unless this is wanted, in which case you would make the event listener only around line 672).
```suggestion
//Self Initialize
window.addEventListener('load', ()=>{
let libretextGlossary = new LibreTextsGlossarizer();
libretextGlossary.makeGlossary();
});
```

Reply:
It looks like you don't get from localStorage at anytime, and the removeGlossary always destroys the key upon initialization.

Reply:
I'll add the event listener to the glossary maker since the sidebar buttons will need to access the glossarizer